### ISSUE TITLE
Octo4a gif cpulimit

### ISSUE THREAD

**Issue by MischaI** [fabianonline/OctoPrint-Telegram#387]
On octo4a(https://github.com/feelfreelinux/octo4a) cant Create gif, cause cpulimit not installed.
How I can install cpulimit on Android?
Or another Solution to Create gifs and Send via telegram?

**Comment by giloser** [comment]
Hi sorry for the delay.
you could try something like this https://github.com/Miliox/android-cpulimit

I don't have a plan for other solution for now the only thing I could think of is to put an option to remove the use of cpulimit but that will be at your own risk as creating the gif could be fatal to print on some device.

**Comment by MischaI** [comment]
Yes, I See, but some android Devices are strong enouth, I think, and I can test this change (without cpulimit), cause my s10 hasnt root, and display broken, so I cant root s10.

**Comment by MischaI** [comment]
Can you make is as Checkbox or Option? 
User should check or uncheck, depend of device.

**Comment by MischaI** [comment]
Any news?
